src/modules/ExternalNotificationModule.cpp

```diff
@@ -68,6 +72,34 @@ bool ascending = true;
 
 #define ASCII_BELL 0x07
 
+<<<<<<< HEAD
+struct ExternalNotificationModule::RGB {
+    constexpr RGB(uint8_t red, uint8_t green, uint8_t blue) : red{red}, green{green}, blue{blue} {}
+
+#define SCALE_ALPHA(COLOR, ALPHA) (uint8_t)((((uint16_t)COLOR) * ALPHA) / (uint16_t)255)
+    constexpr RGB(uint8_t red, uint8_t green, uint8_t blue, uint8_t alpha)
+        : red{SCALE_ALPHA(red, alpha)}, green{SCALE_ALPHA(green, alpha)}, blue{SCALE_ALPHA(blue, alpha)}
+    {
+    }
+#undef SCALE_ALPHA
+=======
+struct ExternalNotificationModule::Color {
+    constexpr Color(uint8_t red, uint8_t green,uint8_t blue)
+    : red{red}, green{green}, blue{blue}
+    {}
+    
+    #define SCALE_ALPHA(COLOR, ALPHA) (uint8_t)((((uint16_t)COLOR) * ALPHA) / (uint16_t)255)
+    constexpr Color(uint8_t red, uint8_t green, uint8_t blue, uint8_t alpha)
+    : red{SCALE_ALPHA(red, alpha)}, green{SCALE_ALPHA(green, alpha)}, blue{SCALE_ALPHA(blue, alpha)}
+    {}
+    #undef SCALE_ALPHA
+>>>>>>> 79093de8 (Fix compilation error due to `rgb` name conflict.)
+
```

Comment:
You somehow checked in a conflicted file. there are merge marks that will bomb during compilation.